neuro_san/client/agent_cli.py

```diff
@@ -166,8 +166,8 @@ def loop_until_done(self, user_input: str, sly_data: Dict[str, Any],
 
             print(f"\nResponse from {state.get('origin_str')}:")
             print(f"{state.get('last_chat_response')}")
-            if state.get("returned_sly_data") is not None:
-                pretty_sly: str = json.dumps(state.get('returned_sly_data'), indent=4, sort_keys=4)
+            if state.get("sly_data") is not None:
+                pretty_sly: str = json.dumps(state.get('sly_data'), indent=4, sort_keys=True)
```

Comment:
Assuming `sly_data` is updated by the server now.
See change in `neuro_san/client/streaming_input_processor.py`